public/pages/Correlations/containers/CreateCorrelationRule.tsx

```diff
@@ -402,7 +430,7 @@ export const CreateCorrelationRule: React.FC<CreateCorrelationRuleProps> = (
 
   const getLogFields = useCallback(
     async (indexName: string) => {
-      return getFieldsForIndex(props.indexService, indexName);
+      return getFieldsForIndex(props.fieldMappingService, indexName);
```

Comment:
This is the only logic change in this file, rest are just formatting changes by prettier